app/views/devise/shared/_password_strength.html.slim

```diff
@@ -10,3 +10,4 @@
       span#pw-strength-txt.bold = '...'
     .h6
       #pw-strength-feedback.italic
+= hidden_field_tag :forbidden_passwords, @forbidden_passwords, id: 'forbidden_passwords'
```

Comment:
Why did we land on a hidden field rather than a data attribute? I'm not sure it matters much, but it feels like an input (even a hidden one) may have a bit more exposure to bots, and I think interacting with data attrs and datasets is maybe a bit nicer.

Reply:
No particular reason. I'm happy to change to a data attribute.

Reply:
I agree with TC, I think a data attribute is easier to query and has a smaller change of unintended side effects.